src/store/flow/block-types/MobilePrimitives_SelectOneResponseBlockStore.ts

```diff
@@ -59,21 +55,13 @@ export const actions: ActionTree<ICustomFlowState, IRootState> = {
   },
 
   deleteChoiceByResourceIdFrom({rootGetters}, {blockId, resourceId}: {blockId: IBlock['uuid'], resourceId: IResource['uuid']}) {
-    const block: ISelectOneResponseBlock = rootGetters['builder/blocksById'][blockId]
-    if (block == null) {
-      throw new ValidationException(`Unable to find block: ${blockId}`)
-    }
-
+    const block: ISelectOneResponseBlock = findBlockWith(blockId, rootGetters['flow/activeFlow']) as ISelectOneResponseBlock
     const choiceKey = String(findKey(block.config.choices, (v) => v === resourceId))
     Vue.delete(block.config.choices, choiceKey)
   },
 
   async reflowExitsFromChoices({dispatch, rootGetters}, {blockId}: {blockId: IBlock['uuid']}) {
-    const block: ISelectOneResponseBlock = rootGetters['builder/blocksById'][blockId]
-    if (block == null) {
-      throw new ValidationException(`Unable to find block: ${blockId}`)
-    }
-
+    const block: ISelectOneResponseBlock = findBlockWith(blockId, rootGetters['flow/activeFlow']) as ISelectOneResponseBlock
```

Comment:
@bzabos , I curious if you didn't have issue on SelectMany when this action `reflowExitsFromChoices` is called by `handleChoiceChanged` or `reflowExitsWhenSwitchingToBranchingTypeNotUnified`?

I'm wondering if we shouldn't use `ISelectOneResponseBlock | ISelectManyResponseBlock` instead of `ISelectOneResponseBlock` only ? (applicable for the whole `as ISelectOneResponseBlock` & `:ISelectOneResponseBlock` iteration)

Reply:
I didn't have any issues! The typing here still works because both have `.choices` property. I think it feels a bit weird to have SelectMany typing in the SelectOne store. Would you be okay if I left it as is?

Reply:
If you don't have issues, let's keep it as it is. Thanks